public/manifest.json

```diff
@@ -18,7 +18,7 @@
       "sizes": "512x512"
     }
   ],
-  "start_url": "",
+  "start_url": ".",
```

Comment:
Just wondering, what does this accomplish?

Reply:
It was the default used by other manifest files, but I think we do actually want the old value here, so I changed it back.